tests/e2e/testRunner.ts

```diff
@@ -39,13 +38,13 @@ const getArg = (argName) => {
 };
 
 let config = defaultConfig;
-const setConfigPath = (configPathParam) => {
+const setConfigPath = (configPathParam: string | undefined): void => {
```

Comment:
No need to specify `void` type
```suggestion
const setConfigPath = (configPathParam: string | undefined) => {
```